BROWSER_README.md

```diff
@@ -0,0 +1,99 @@
+_This is one of the steps in the Truss interview process. If you've
+stumbled upon this repository and are interested in a career with
+Truss, [check out our jobs page](https://truss.works/jobs)._
+
+# Truss Software Engineering Interview
+
+## Introduction and expectations
+
+Hi there! Please complete the problem described below to the best of
+your ability, using the tools you're most comfortable with. Assume
+you're sending your submission in for code review from peers;
+we'll be talking about your submission in your interview in that
+context.
+
+We expect this to take less than 4 hours of actual coding time. Please
+submit a working but incomplete solution instead of spending more time
+on it. We're also aware that getting after-hours coding time can be
+challenging; we'd like a submission within a week and if you need more
+time please let us know.
+
+Approach this solution the way you would a real world problem. Use
+libraries where it makes sense to, and be prepared to explain your
+thought process if you do.
+
+We will review your solution in a recent version of Firefox, Chrome, or Safari.
+
+If you have any questions, please contact [hiring@truss.works](mailto:hiring@truss.works); we're
+happy to help if you're not sure what we're asking for or if you have
+questions.
+
+## How to submit your response
+
+Please submit your solution by emailing [hiring@truss.works](mailto:hiring@truss.works) a link to **one** of the following:
+
+-   A public git repository (Github is fine) that contains your code and a `README.md` that tells us how to build and run it.
+-   Your solution running in an online IDE such as [CodePen](https://codepen.io) or [Glitch](https://glitch.com).
+
+Please ensure that your submission includes everything needed to run and view your project:
+
+-   If your solution code requires compilation (such as with a tool like webpack or create-react-app),
+    please provide as a part of your submission complete instructions for running your code, including
+    a `package.json` file with all dependencies included.
```

Comment:
Will these instructions will vary by platform? For example if a candidate is developing on linux but we are running their code on macOS, will the supplied instructions for installing dependencies work?

Reply:
it is probably worth saying we'll be running their solutions on MacOS but they can specify a node version for example